lecture_2/homework/app/controllers/clans/warriors_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Clans
+  class WarriorsController < ApplicationController
+    def show
+      render json: warrior.as_json(include: 'clan', include: 'defensible')
+    end
+
+    def index
+      warriors = clan.warriors
+      if params.key?(:alive)
+        if params[:alive].to_i == 0
+          render json: warriors.dead.to_json
```

Comment:
`render json:` sugeruje railsom, że będziemy chcieli json. Jeśli przekażemy mu obiekt to automatycznie wywoła na nim `.to_json`, więc możemy to spokojnie pominąć 😃